src/jit/emitxarch.cpp

```diff
@@ -1494,16 +1494,15 @@ inline UNATIVE_OFFSET emitter::emitInsSizeRR(instruction ins, regNumber reg1, re
     emitAttr size = EA_SIZE(attr);
 
     UNATIVE_OFFSET sz;
-#ifdef _TARGET_AMD64_
-    // If Byte 4 (which is 0xFF00) is non-zero, that's where the RM encoding goes.
+
+    // If Byte 4 (which is 0xFF00) is zero, that's where the RM encoding goes.
```

Comment:
@CarolEidt The original comment said that if Byte 4 (at mask 0xFF00) is zero, then the size is 5. But that's obviously the opposite of what the code does. My comment change actually reflects what the code does. And I think it reflects that comment in instrsxarch.h as well: in the RM encoding column, we leave the RM slot zero *except* for the case of 4-byte opcodes that need RM, where all the bytes are already filled with opcode. In that case, we need some special code to determine the RM version size.

It looks like this code is basically a hack for handling the case of a 5-byte opcode + RM in the table, where we don't actually have 5 bytes in the table.

Am I still confused?